### DISCUSSION TITLE
PSBZS A1 - Lidl Parkside smart watering timer -> are more options possible?

### DISCUSSION THREAD

**Discussion by nukleuz80** [Koenkk/zigbee2mqtt#8274] [Devices]
Hi there,

is it possible to get more than the available options?
https://www.zigbee2mqtt.io/devices/PSBZS_A1.html

What about battery and frost protection?

Thanks!

**Comment by Pferdebockwurst** [comment]
This is fixed with [commit #5710](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5710).
